cvat-ui/src/components/create-task-page/create-task-content.tsx

```diff
@@ -102,26 +102,42 @@ const defaultState: State = {
 };
 
 const UploadFileErrorMessages = {
-    one: 'It can not be processed. You can upload an archive with images, a video, a pdf file or multiple images',
-    multi: 'It can not be processed. You can upload one or more videos',
+    one: 'It can not be processed. You can upload an archive with images, a video, a pdf file or multiple images. ',
+    multi: 'It can not be processed. You can upload one or more videos. ',
 };
 
+function fileExtensionList(files: RemoteFile[]): string {
```

Comment:
I would suggest this function to return string[] with extensions, and call it ``receiveExtensions``
For a file without extension it will return the file name instead of extension, probably it is not correct